src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -423,7 +426,7 @@ public void userProfileCreateUserWireMock(HttpStatus status) {
                 + "  \"idamMessage\": \"\""
                 + "  },"
                 + " {"
-                + "  \"userIdentifier\":\"%s"  + "\","
+                + "  \"userIdentifier\":\"%s" + "\","
                 + "  \"firstName\": \"adil\","
                 + "  \"lastName\": \"oozeerally\","
                 + "  \"email\": \"adil.ooze@hmcts.net\","
```

Comment:
Please check the policy about using actual email id's and names.

Reply:
This is not my actual email id, our names however have been there for more than a year, but as previously stated - happy to change if required

Reply:
Let me check and come back to you. We can change this later as well.